static/ankersrv.css

```diff
@@ -0,0 +1,21 @@
+.nav-link {
+    color: #88f387!important;
```

Comment:
Using a lot of `!important` is usually a sign that we are not quite hitting the right css scopes. Are they really required? As long as our rule has the same css specificity, and our css is loaded after bootstrap, it should override it without problems.

At the very least, please put a space before `!important` :smile: